app/javascript/mastodon/utils/scrollbar.ts

```diff
@@ -16,20 +12,20 @@ const getActualScrollbarWidth = () => {
   outer.appendChild(inner);
 
   const scrollbarWidth = outer.offsetWidth - inner.offsetWidth;
-  outer.parentNode.removeChild(outer);
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  outer.parentNode!.removeChild(outer);
```

Comment:
[`Element.remove()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/remove) exists now, but I'm hesitant to change behavior in a TypeScript conversion PR. I've learned the hard way that that's how I tend to introduce bugs 😄

Reply:
Support is very wide, you can use it and it will be cleaner :)

Reply:
Awesome thanks 🙂 

Just confirming - for this and other runtime fixes, would you prefer them in this PR or a separate one? I personally prefer separating type system changes from runtime but also understand the extra overhead isn't ideal.

Reply:
I am fine with doing it in the same PR, as long as you add a comment pointing those changes